_data/projects/blockstreet.yml

```diff
@@ -0,0 +1,12 @@
+name: Blockstreet
+desc: Blockstreet aims to become the go-to source for the explanation of complex topics for users that are new to the blockchain ecosystem.
+site: https://blockstreet.io
+tags:
+- javascript
+- vuejs
+- nodejs
+- cryptocurrency
+- blockchain
+upforgrabs:
+  name: help needed
```

Comment:
This value should be the name of a label assigned to issues that are suitable for new contributors to pick up